robot_ws/src/object_tracker_robot/scripts/run_turtlebot_controller.sh

```diff
@@ -2,5 +2,9 @@
 
 set -ex
 
+DOWNLOAD_MODEL_PYTHON_PATH="$(dirname "$0")/download_model.py"
+CONFIG_YAML_PATH="$(dirname "$0")/../../share/object_tracker_robot/config/model_config.yaml"
 MODEL_PATH="$(dirname "$0")/../../share/object_tracker_robot/model.pb"
+
 python3 -m robomaker.inference_worker ${MODEL_PATH}
+python ${DOWNLOAD_MODEL_PYTHON_PATH} ${CONFIG_YAML_PATH} ${MODEL_PATH}
```

Comment:
Do you need both Python 2 and 3 for this script to work?

Python 3 for `python3 -m robomaker.inference_worker ${MODEL_PATH}` and Python 2 for `python ${DOWNLOAD_MODEL_PYTHON_PATH} ${CONFIG_YAML_PATH} ${MODEL_PATH}`?

Reply:
We have removed run_turtlebot_controller.sh as per @AAlon suggested and make inference_worker as a node and passed arguments from launch. Also, we have converted download_model.py into Python3 (so, no dependency on Python2)

Please find [PR#41](https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/pull/41) for more details.